coordinator/clientapi/clientapi.go

```diff
@@ -139,8 +139,18 @@ func (a *ClientAPI) GetCertQuote(ctx context.Context) (cert string, certQuote []
 		return "", nil, errors.New("pem.EncodeToMemory failed for intermediate certificate")
 	}
 
+	// Get existing quote for root cert, or generate a new one over provided nonce
+	var reportData []byte
+	if nonce != nil {
```

Comment:
```suggestion
	if len(nonce) > 0 {
```
In most cases, don't treat nil and empty slices differently. (See also https://github.com/edgelesssys/wiki/blob/master/company/gostyle.md#nil-is-a-valid-slice)